uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar.component.js

```diff
@@ -41,6 +41,7 @@ define([
     vm.showContactName = false;
     vm.showFilters = false;
     vm.supportData = {};
+    vm.userpermi = 'staff';
```

Comment:
Could you please fix this one and add a test.

Reply:
Ah sorry, I added a test for that:
https://github.com/civicrm/civihr/pull/2656/files#diff-333e6c3beacc230ca218ee76bc1006c3R124

The issue is that this property is overridden as soon as the controller is initialized. This definition is only here as a reference. Is it ok just to fix the name?